lib/spree/shipment_notice.rb

```diff
@@ -29,6 +44,7 @@ def update
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
+        shipment.reload.ready! unless shipment.ready?
```

Comment:
Ah sorry I didn't realize this was part of the `update` method, in the diff it appeared this was part of the `locate` method. This is fine :)